q42stats/build.gradle

```diff
@@ -9,12 +9,11 @@ plugins {
 group = 'com.github.Q42'
 
 android {
-    compileSdkVersion 30
-    buildToolsVersion "30.0.2"
+    compileSdkVersion compile_sdk_version
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 30
+        minSdkVersion min_sdk_version
+        targetSdkVersion compile_sdk_version
         versionCode 1
         versionName "0.0.3"
```

Comment:
versionName should be "1.0" to indicate that it's not meaningful

That it really matters since it is a sample app